### PR TITLE
"Getting Started" Doc Updates

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
The "Getting Started" section is quite updated. Here is some up to date documentation.

### CODE REVIEW

File: src/GettingStarted/Dependencies.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Dependencies
+All Terra UI components have the following peer dependencies:
 
-- [react](https://github.com/facebook/react/tree/master/packages/react) - Javascript Library for building UI.
-- [react-dom](https://github.com/facebook/react/tree/master/packages/react-dom) - React package for working with the DOM.
+- [react](https://reactjs.org/docs/react-api.html#overview) - Javascript Library for building user interfaces.
+- [react-dom](https://reactjs.org/docs/react-dom.html#overview) - React package for working with the DOM.
 
-If you see `UNMET PEER DEPENDENCY` errors when installing terra components, you should manually install the peer dependencies listed in your console.
+Some components may have additional peer dependencies and these will be listed in their package.json. If `UNMET PEER DEPENDENCY` errors are shown when installing terra components, then manually install the peer dependencies listed in the console.
```
Comment (ChaseSinclair) [inline src/GettingStarted/Dependencies.md]: Missing a comma after "peer dependencies"

File: src/GettingStarted/WhatIsTerra.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+Terra offers a set of configurable React components designed to help build scalable and modular healthcare focused applications. These components follow Cerner’s Design Standard Guidelines and are well-maintained and tested. The User Interface (UI) framework easily integrates with webpack-based workflows and was created to solve real-world issues in projects we work on day to day.
 
-Getting familiar with [React](http://facebook.github.io/react/) before getting started with Terra UI is recommended but not required. Terra UI is a set of React components, so understanding how React fits into web development is important. Terra UI components work with React 15.4.2+.
+It is highly recommended to become familiar with [React](https://reactjs.org/) before using Terra UI, but it is not required. Since Terra UI is a set of React components, it is important to understanding how React fits into web development. Terra UI components work with React v16.2.0+.
```
Comment (ChaseSinclair) [inline src/GettingStarted/WhatIsTerra.md]: Should this be "it is important _in_ understanding how React..." or "it is important _to_ understand how React..."?

File: src/GettingStarted/Features.md [hunk 3]
```diff
@@ -1,23 +1,22 @@
 ## Features
+Terra UI have been designed as modular components with consistent design standards, responsive web design, and encapsulated and configurable styling.
```
Comment (tbiethman) [inline src/GettingStarted/Features.md]: Do we need this sentence, or can we flow right into the updated feature list?

File: src/GettingStarted/WhatIsTerra.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
 ## What is Terra?
-Terra offers a set of configurable React components designed to help build scalable and modular healthcare focused applications. These components follow Cerner’s Design Standard Guidelines and are well-maintained and tested. The UI framework easily integrates with webpack-based workflows and was created to solve real-world issues in projects we work on day to day.
+Terra offers a set of configurable React components designed to help build scalable and modular healthcare focused applications. These components follow Cerner’s Design Standard Guidelines and are well-maintained and tested. The User Interface (UI) framework easily integrates with webpack-based workflows and was created to solve real-world issues in projects we work on day to day.
```
Comment (tbiethman) [inline src/GettingStarted/WhatIsTerra.md]: Nit: "well-maintained" sounds a little presumptuous :sweat_smile: Something like "actively maintained" would sound less biased.

Comment (emilyrohrbough) [reply src/GettingStarted/WhatIsTerra.md]: Agreed!

File: src/GettingStarted/Dependencies.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ## Dependencies
+All Terra UI components have the following peer dependencies:
 
-- [react](https://github.com/facebook/react/tree/master/packages/react) - Javascript Library for building UI.
-- [react-dom](https://github.com/facebook/react/tree/master/packages/react-dom) - React package for working with the DOM.
+- [react](https://reactjs.org/docs/react-api.html#overview) - Javascript Library for building user interfaces.
+- [react-dom](https://reactjs.org/docs/react-dom.html#overview) - React package for working with the DOM.
```
Comment (mjhenkes) [inline src/GettingStarted/Dependencies.md]: Should we list terra-base?

**Pull request merged**